pkg/ddl/backfilling_dispatcher.go

```diff
@@ -42,33 +42,33 @@ import (
 	"go.uber.org/zap"
 )
 
-// BackfillingDispatcherExt is an extension of litBackfillDispatcher, exported for test.
-type BackfillingDispatcherExt struct {
+// BackfillingSchedulerExt is an extension of litBackfillScheduler, exported for test.
+type BackfillingSchedulerExt struct {
 	d          *ddl
 	GlobalSort bool
 }
 
-// NewBackfillingDispatcherExt creates a new backfillingDispatcherExt, only used for test now.
-func NewBackfillingDispatcherExt(d DDL) (dispatcher.Extension, error) {
+// NewBackfillingSchedulerExt creates a new backfillingSchedulerExt, only used for test now.
+func NewBackfillingSchedulerExt(d DDL) (scheduler.Extension, error) {
 	ddl, ok := d.(*ddl)
 	if !ok {
 		return nil, errors.New("The getDDL result should be the type of *ddl")
 	}
-	return &BackfillingDispatcherExt{
+	return &BackfillingSchedulerExt{
 		d: ddl,
 	}, nil
 }
 
-var _ dispatcher.Extension = (*BackfillingDispatcherExt)(nil)
+var _ scheduler.Extension = (*BackfillingSchedulerExt)(nil)
 
-// OnTick implements dispatcher.Extension interface.
-func (*BackfillingDispatcherExt) OnTick(_ context.Context, _ *proto.Task) {
+// OnTick implements scheduler.Extension interface.
+func (*BackfillingSchedulerExt) OnTick(_ context.Context, _ *proto.Task) {
 }
 
 // OnNextSubtasksBatch generate batch of next step's plan.
-func (dsp *BackfillingDispatcherExt) OnNextSubtasksBatch(
+func (dsp *BackfillingSchedulerExt) OnNextSubtasksBatch(
```

Comment:
```suggestion
func (se *BackfillingSchedulerExt) OnNextSubtasksBatch(
```
change the receiver name